index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+      <header>JS Documentation</header>
+
+        <ul id="ul-navbar">
+          <li> 
+            <a class="nav-link" href="#Introduction">
+            Introduction
+            </a>
+          </li>
+
+          <li>
+            <a class="nav-link" href="#What_you_should_already_know">
+            What you should already know
+            </a>
+          </li>
+
+          <li>
+            <a class="nav-link" href="#JavaScript_and_Java">
+            JavaScript and Java
+            </a>
+          </li>
+
+          <li>
+            <a class="nav-link" href="#Hello_world">
+            Hello world
+            </a>
+          </li>
+
+          <li>
+            <a class="nav-link" href="#Variables">
+            Variables
+            </a>
+          </li>
+
+          <li>
+            <a class="nav-link" href="#Declaring_variables">
+            Declaring variables
+            </a>
+          </li>
+        </ul>
+
+    </nav>
+
+    <main id="main-doc">
+
+      <section class="main-section" id="Introduction">
+        
+          <header>Introduction</header>
```

Comment:
Please use h1 or h2 tag instead of the header tag. It's a good practice to use a header tag to represent a container for introductory content like navigations and links.